docs/examples/all-ligand-explicit.rst

```diff
@@ -0,0 +1,184 @@
+.. _all-ligand-explicit:
+
+Absolute Binding of Binders and Non-binders to T4 lysozyme L99A in Explicit Solvent with Combinatorial Options
+==============================================================================================================
+
+This example looks at the absolute binding of a series of binders and non-binder to the T4 Lysozyme L99A protein.
```

Comment:
"looks at the absolute binding of a series of binders and non-binder" => "performs absolute binding free energy calculations for a series of small molecules that contain known binders and nonbinders"